File: Day-9/atm.cpp
#include <iostream>
using namespace std;

int main() {
	int withdraw_amt;
	float balance = 0.00;
    cout << "Enter the Withdraw Amount and Balance: ";
	cin >> withdraw_amt >> balance;
	if(withdraw_amt + 0.5 > balance || withdraw_amt % 5 != 0)
	    printf("%.2f", balance);
	else
	    printf("%.2f", balance - withdraw_amt - 0.5);
	return 0;
}

File: Day-9/atm.py
withdraw_amt, balance = map(float, input("Enter the Withdraw Amount and Balance: ").split())
if (withdraw_amt + 0.5 > balance) or (withdraw_amt % 5 != 0):
    print("{:.2f}".format(balance))
else:
    print("{:.2f}".format(balance - withdraw_amt - 0.50))